spec/sec-operations-on-objects-patch.html

```diff
@@ -0,0 +1,28 @@
+<emu-clause id="sec-operations-on-objects">
+  <h1>Operations on Objects</h1>
+  <ins>
+    <emu-clause id="sec-getoption" aoid="GetOption">
+      <h1>GetOption ( _options_, _property_, _type_, _possibleValues_, _fallback_ )</h1>
```

Comment:
i still think possibleValues should be last here, and marked optional.

Reply:
I'd like the signature of the abstract operation to align with the one in ECMA-402, since it is likely they would merely point to this version in the future. I'd like to get @littledan's input on that, however.

Reply:
I don’t think it’s appropriate for 402 choices, unreviewed by 262 editors, to constrain 262 - i think 402 can adapt to whatever changes we make here.

cc @leobalter

Reply:
If I do change it, then I'll likely change the name as well or just inline it so as to not conflict with the ECMA-402 version.

Reply:
It definitely can't be inlined; it's setting the first precedent in the language for an options object. The changes I'm suggesting don't conflict with 402's version semantically, just editorially, and since 402 is layered on top of 262, it's appropriate for them to either shadow GetOption, or adapt to 262's - and I don't think renaming it to something less clear is a good precedent to set either.

Reply:
Precedent *ish*, I would argue that proxy handlers are something like an options argument as well, as are property descriptors passed to `Object.defineProperty`. As a matter of fact, `ToPropertyDescriptor` could probably be rewritten in terms of `GetOption` as well.